library/core/src/iter/traits/iterator.rs

```diff
@@ -3414,10 +3414,9 @@ pub trait Iterator {
     /// ```
     #[stable(feature = "iter_copied", since = "1.36.0")]
     #[rustc_diagnostic_item = "iter_copied"]
-    fn copied<'a, T: 'a>(self) -> Copied<Self>
+    fn copied<'a, T: Copy + 'a>(self) -> Copied<Self>
     where
         Self: Sized + Iterator<Item = &'a T>,
```

Comment:
I did mean for the comment I made about `where` clauses to generalize, because the `where` is not easy to elide here.

Reply:
Thank you. I realize that but I just prefer that if we're moving them anywhere we move them towards the `where`, generally.